profile/README.md

```diff
@@ -0,0 +1,80 @@
+# Wagtail Nest
+
+Wagtail Nest houses community maintained third-party packages for Wagtail CMS.
+
+Many third-party Wagtail packages are created and maintained by a single person. This is a risk. Too often single maintainer packages become abandon-ware. The maintainer just doesn’t have the time or motivation to update and release the package.
+
+Wagtail Nest aims to address this problem by housing and maintaining packages collaboratively. Packages will be transferred into the wagtail-nest GitHub organisation. Members of the Wagtail Nest collective will collaboratively maintain and upgrade packages.
+
+By moving the package to Wagtail Nest, the current maintainer (or original author) agrees to collaborate with the other members and will seek consensus when it comes to contentious changes.
+
+
+## Guidelines
+
+- Be nice
+- The primary goal is maintenance. New features are a nice to have
+- Release often, every time updates are done.
+- Use [SemVer](https://semver.org/). If there are breaking changes, bump the major version.
+- Test against Wagtail main, [nightly](https://github.com/wagtail/wagtail/wiki/Nightly-testing-of-official-plugins).
```

Comment:
At first glance it’s not clear to me whether you mean testing against the `main` branch, or the latest nightly release, or both. Could be worth clarifying?